ccdproc/ccddata.py

```diff
@@ -98,21 +108,21 @@ class CCDData(NDDataArray):
     This is useful, for example, when plotting a 2D image using
     matplotlib.
 
-        >>> from ccdproc import CCDData   
+        >>> from ccdproc import CCDData
         >>> from matplotlib import pyplot as plt   # doctest: +SKIP
-        >>> x = CCDData([[1,2,3], [4,5,6]], unit='adu') 
+        >>> x = CCDData([[1,2,3], [4,5,6]], unit='adu')
         >>> plt.imshow(x)   # doctest: +SKIP
 
     """
     def __init__(self, *args, **kwd):
         if 'meta' not in kwd:
             kwd['meta'] = kwd.pop('header', None)
         if 'header' in kwd:
-            raise ValueError("Can't have both header and meta")
+            raise ValueError("Can't have both header and meta.")
```

Comment:
That's a good point, I'll scan through the `raise`-items again.

How about `log`-infos? I think they are printed:

```
>>> nd2 = NDData(q, unit=u.cm)  
INFO: Overwriting Quantity's current unit with specified
```
So lowercase them too?